scripts/prepare_test_env.sh

```diff
@@ -19,9 +19,14 @@ export PROXY=$2
 export DEX_DOMAIN_NAME="dex.$DOMAIN_NAME"
 export MGMT_DOMAIN_NAME="mgt.$DOMAIN_NAME"
 
+sudo pip install -r ./requirements.txt
+sudo pip install -r ../tests/requirements.txt
+
 echo "Fetching CA for $MGMT_DOMAIN_NAME"
-./get_cert.sh $MGMT_DOMAIN_NAME ca-ing $PROXY > ca.pem
-cat ./ca.pem
+# ./get_cert.sh $MGMT_DOMAIN_NAME ca-ing $PROXY > ca.pem
```

Comment:
If you apply changes in exporting MGMT_DOMAIN_NAME in CI configuration, it will work